app/qml/components/MMNotification.qml

```diff
@@ -38,15 +38,15 @@ Rectangle {
     anchors.verticalCenter: parent.verticalCenter
     anchors.left: parent.left
     anchors.leftMargin: 20 * __dp
-    size: model.icon === MM.NotificationType.Check ? __style.icon18 : __style.icon24
```

Comment:
I think it has size.icon16

Reply:
by the design, icon is 24px with margins. 18px without margins.
16px is not good readable

Reply:
by design document, icon with margins are 24px, without margins (circle) are 18px.
16px are not good readable

Reply:
in that case change in mmstyle.h icon16 -> icon18